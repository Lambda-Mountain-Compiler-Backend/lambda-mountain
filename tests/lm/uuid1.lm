
_uuid := ();
uuid := Î». (tail(
   (set _uuid (inc _uuid))
   (clone-rope ('_uuid_ (dump-i _uuid)))
));

main := (
   (print-s (uuid()))
   (print-s (uuid()))
);
