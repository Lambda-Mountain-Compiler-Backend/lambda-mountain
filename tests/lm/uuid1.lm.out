uuid_0000000000000001uuid_0000000000000002
